index.html

```diff
@@ -0,0 +1,251 @@
+          <h4>Lecture</h4>
+          <p>
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Placeat
+            consequuntur ut,
+        </p>
+        </article>
+        <article class="tab">
+          <img src="Icons/icon2.png" alt="" />
+          <h4>Exhibition</h4>
+          <p>
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Placeat
+            consequuntur ut,
+          </p>
+        </article>
+        <article class="tab">
+          <img src="Icons/icon3.png" alt="" />
+          <h4>Forum</h4>
+          <p>
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Placeat
+            consequuntur ut,
+          </p>
+        </article>
+        <article class="tab">
+            <img src="Icons/icon4.png" alt="" />
+          <h4>Workshop</h4>
+          <p>
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Placeat
+            consequuntur ut,
+        </p>
+        </article>
+        <article class="tab">
+          <img src="Icons/icon5.png" alt="" />
+          <h4>Worrior Ignite</h4>
+          <p>
+              Lorem ipsum dolor sit amet consectetur, adipisicing elit. Placeat
+              consequuntur ut,
+            </p>
+        </article>
+        <div >
+            <button class="mainProgramButton">Join Worrior Global Champions 2023</button>
+        </div>
+      </div>
+      <span class="wholeProgram"><a href="#">SEE THE WHOLE PROGRAM</a></span>
+    </section>
+
+    <section class="Featured">
+      <h3>Feature Speakers</h3>
+      <div class="Speakers" id="Speakers"></div>
+      <span class="MoreSpeakers" id="MoreSpeakers" >
+          <div class="Speakers" id="otherSpeakers"></div>
+      </span>
+      <button class="ArrowDownButton" id="ArrowDownButton">More <img src="Icons/arrow-down.png" alt=""></button>
+    </section>
+
+    <section class="Partners">
+        <h2>Partners</h2>
+        <div>
+            <img src="Icons/google.png" alt="">
+            <img src="Icons/addidas.png" alt="">
+            <img src="Icons/amazon.png" alt="">
+            <img src="Icons/instagram.png" alt="">
+            <img src="Icons/volswagon.png" alt="">
+        </div>
+    </section>
+    
+    <footer class="footer">
+        <div class="mitMain">
+            <img src="Icons/mit.png" alt="">
+            <div>
+                <span>MIT</span>
+                <span>Licensed</span>
+            </div>
+        </div>
+        <div class="footerDescripation">
+            <span class="footer-span">2020 MIT License was the most popular software license found</span>
+            <P class="footer-descripation-disktop footer-descripation-mobile">The MIT License is a permissive free software license originating at the Massachusetts Institute of Technology in the late 1980s.As a permissive license,
+            it puts only very limited restriction on reuse and has, therefore, high license compatibility.</P>
+        </div>
+    </footer>
```

Comment:
To make our page more semantic, kindly take the footer section away from the main section😉 Your page structure should have the `header`, `main` and `footer` and they should stand independently😉 but currently, your `footer` section is within the `main` section. kindly fix this issue😊😊 kindly follow this structure for both your `index` and `about` page.😊😊